cmd/sovereignnode/systemTestDemo/README.md

```diff
@@ -95,7 +95,7 @@ triggers the transmission of blocks, starting from an arbitrary nonce, with inco
 Each event includes the transfer of an NFT and an ESDT token. The periodic transmission consists of 2 NFTs
 (ASH-a642d1-01 & ASH-a642d1-02) and one ESDT token (WEGLD-bd4d79).
 
-To verify the success of token transfers, you can utilize the sovereign proxy for the subscribed address, which can be
+To verify the success of token transfers, you can utilize the sovereign proxy for the subscribed address where identifier = "deposit", which can be
```

Comment:
This file needs to be reformatted. Line passed 120 char limits. After reformat, all lines need to be re-arranged